### ISSUE TITLE
Request: `randomize (all ducks) size`

### ISSUE THREAD

**Issue by mikeharv** [code-dot-org/dance-party#183]
This is not a bug so mark it low priority. Beginning in level 11, students get a block to randomize a sprite's property. This is great for `when up pressed` or `every 2 measures`. It feels like it's missing from the set of group commands. I also wonder if we want to be able to `set all ducks tint to blue`.

**Comment by boatmarina** [comment]
good idea.  Pushing this to Post-HOC milestone as a great enhancement idea

**Comment by joshlory** [comment]
One possible conflict: Hadi wants the `"all" do "(random)" dance` to make all of the actors do the _same_ random dance.  This might be at odds conceptually with setting each dancer to a _different_ random location.

**Comment by mikeharv** [comment]
@joshlory Could the `randomize "all" (property)` block work the same as Hadi's suggestion (set all to the same random size/rotation/tint) but we just leave off location? Users can already randomize all locations using the layout blocks.

**Comment by mikeharv** [comment]
[block definition](https://levelbuilder-studio.code.org/pools/Dancelab/blocks/Dancelab_setPropRandomEach/edit)
